src/frontends/tensorflow/src/op/squeeze.cpp

```diff
@@ -15,7 +15,7 @@ namespace op {
 
 OutputVector translate_squeeze_op(const NodeContext& node) {
     auto input = node.get_input(0);
-    auto axes = node.get_attribute<std::vector<int64_t>>("squeeze_dims");
+    auto axes = node.get_attribute<std::vector<int64_t>>("squeeze_dims", {});
```

Comment:
Where does this attribute come from? I have not found it here: https://www.tensorflow.org/api_docs/python/tf/raw_ops/Squeeze

Reply:
https://www.tensorflow.org/api_docs/python/tf/compat/v1/squeeze
Looks like we also need to check axis attribute, squeeze_dims has been deprecated
![image](https://user-images.githubusercontent.com/21181928/186343835-7bd2b6f4-d92e-4cf4-9a03-bd23020a2d11.png)